src/main/java/view/AssistedView.java

```diff
@@ -1,4 +1,153 @@
 package view;
 
-public class AssistedView {
+import interface_adapter.assisted_mode.AssistedModeViewModel;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.net.URL;
+
+import static interface_adapter.assisted_mode.AssistedModeColourConstants.TABLECOLOUR;
+
+public class AssistedView extends JPanel{
+
+    private final AssistedModeViewModel assistedModeViewModel;
+
+    public AssistedView(AssistedModeViewModel assistedModeViewModel) {
+        this.assistedModeViewModel = assistedModeViewModel;
+
+        setBackground(TABLECOLOUR);
+
+        // working with GridLayout(), so here are filler panels with the right background colour to just help with
+        // layout issues.
+        JPanel fillerTopPanel = new JPanel();
+        fillerTopPanel.setBackground(TABLECOLOUR);
+
+        setLayout(new GridLayout(1, 3));
+        Font font = new Font("Times New Roman", Font.BOLD, 15);
```

Comment:
I'd suggest creating a constant font to use to avoid magic values. There are other magic values in this file and I'm not gonna comment on all of them, so you if you do decide to make constants just look through the whole file to see what you need.